### PR TITLE
fix: update @babel/preset-env to 2.13.x

### PR DESCRIPTION
**Pull request by pi0**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

#8882 #8883 

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Comment by SirWindfield** [comment]
Is this the PR that is meant to include the functions that were present in the utils file?

**Comment by manniL** [comment]
> 
> 
> Is this the PR that is meant to include the functions that were present in the utils file?

Nope, that happened in https://github.com/nuxt/nuxt.js/pull/8883 :)

**Comment by clarkdo** [comment]
@pi0 Are you still looking into this? Do you want me to have a look?

**Comment by pi0** [comment]
@clarkdo Sure. Feel free selfassigning. Made it as a placeholder to keep track

**Comment by clarkdo** [comment]
@pi0 `unknown polyfill "es6.array.slice"` has been fixed in https://github.com/babel/babel/pull/12870 since babel v7.13.5, I pushed a temp fix for another missing polyfill error `unknown polyfill "web.dom.iterable"` in compat-data

**Review by clarkdo — DISMISSED** [review]

**Comment by clarkdo** [comment]
@pi0 Can you give me a review on this ?

File: packages/babel-preset-app/src/index.js [hunk 1]
```diff
@@ -87,6 +87,12 @@ module.exports = (api, options = {}) => {
     corejs = { version: Number(corejs) }
   }
 
+  // TODO: remove this when babel-plugin-polyfill-corejs2 fixes web.dom.iterable injection
+  if (corejs.version === 2) {
+    const corejs2BuiltIns = require('@babel/compat-data/corejs2-built-ins')
+    corejs2BuiltIns['web.dom.iterable'] = {}
+  }
```
Comment (nicolo-ribaudo) [inline packages/babel-preset-app/src/index.js:94]: Fixed in `babel-plugin-polyfill-corejs2@0.1.6`, sorry!

**Comment by pi0** [comment]
Tests seem passing. Thanks so much @nicolo-ribaudo @clarkdo for helping on this 💯 

Merging to try on nuxt-edge. (releasing by 2.16 to ensure everything is stable)

**Pull request merged**